scout/server/blueprints/institutes/templates/overview/cases.html

```diff
@@ -192,6 +192,54 @@
   </tr>
 {% endmacro %}
 
+{% macro sanger_to_be_validated(unevaluated, validated_by_others) %}
+ <div class="row container alert alert-info alert-dismissible" role="alert">
+  <div class="col-1">
+   <button type="button" class="close" data-bs-dismiss="alert" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+  </div>
+  <div class="col">
+    You have <strong>{{ unevaluated|length }}</strong> cases with Sanger validations to evaluate.
+    {% if validated_by_others == true %}
+      These variants have been validated by another user but they need also your validation since you ordered the Sanger sequencing.
```

Comment:
```suggestion
      These variants have been validated by another user, but you ordered the Sanger sequencing. Adding your validation will silence this message.
```
I'm a little uncomfortable by this being required and adding work for ppl not really requesting the feature, but I guess its a reasonable compromise.

Reply:
Great suggestion, I'm sure somebody would have complained for sure otherwise!